sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/FunctionCallingSample.java

```diff
@@ -46,62 +48,65 @@ public static void main(String[] args) {
             .buildClient();
 
         List<FunctionDefinition> functions = Arrays.asList(
-            new FunctionDefinition("getCurrentWeather")
-                .setDescription("Get the current weather")
-                .setParameters(BinaryData.fromObject(getFunctionDefinition()))
+                new FunctionDefinition("getCurrentWeather")
+                        .setDescription("Get the current weather")
+                        .setParameters(BinaryData.fromObject(getCurrentWeatherFunctionParameters()))
         );
+        List<ChatRequestMessage> chatRequestMessages = new ArrayList<>();
+        chatRequestMessages.add(new ChatRequestUserMessage("What should I wear in Boston depending on the weather?"));
 
-        List<ChatRequestMessage> chatMessages = new ArrayList<>();
-        chatMessages.add(new ChatRequestUserMessage("What should I wear in Boston depending on the weather?"));
-
-        ChatCompletionsOptions chatCompletionOptions = new ChatCompletionsOptions(chatMessages)
-            .setFunctionCall(FunctionCallConfig.AUTO)
-            .setFunctions(functions);
-
-        ChatCompletions chatCompletions = client.getChatCompletions(deploymentOrModelId, chatCompletionOptions);
-        List<ChatRequestMessage> chatMessages2 = handleFunctionCallResponse(chatCompletions.getChoices(), chatMessages);
+        ChatCompletions chatCompletions = client.getChatCompletions(deploymentOrModelId,
+                new ChatCompletionsOptions(chatRequestMessages)
+                        .setFunctionCall(FunctionCallConfig.AUTO)
+                        .setFunctions(functions));
```

Comment:
This sample is about `FunctionCall` and it's correctly named. I think the ones below need to be updated to reflect the difference between a `tool_call` and a `function_call`.